Added support for credentials without cryptographic holder binding (#210)
openid/OpenID4VC-HAIP: openid4vc-high-assurance-interoperability-profile-1_0.md

* Combined Issuance of IETF SD-JWT VC and ISO mdoc
* Both issuer-initiated and wallet-initiated issuance
* Presentation and Issuance of PID and (Q)EAA as defined in Architecture and Reference Framework [@EU.ARF] implementing [@eIDAS2.0].

> * Issuance and presentation of credentials with and without cryptographic holder binding

Reviewer: Cryptographic Key Binding is the term we use in VCI:

```suggestion
* Issuance and presentation of credentials with and without cryptographic key binding
```

> * Issuance and presentation of credentials with and without cryptographic holder binding

Reviewer 2: ```suggestion
* Issuance and presentation of Credentials with and without cryptographic holder binding
```

### Key Attestation {#key-attestation}

> Wallets MUST support key attestations as defined in Annex D of [@!OIDF.OID4VCI]. If batch issuance is used and the newly issued credential instances shall be bound to a public key for cryptograohic holder binding, all public keys used in Credential Request SHOULD be attested within a single key attestation.

Reviewer: I think we can be a clearer that it is the issuer that decides this:

```suggestion
Wallets MUST support key attestations as defined in Annex D of [@!OIDF.OID4VCI]. If batch issuance is used and the Credential Issuer has indicated (via `cryptographic_binding_methods_supported ` in it's metadata) that cryptographic binding is required, all public keys used in Credential Request SHOULD be attested within a single key attestation.
```

> Wallets MUST support key attestations as defined in Annex D of [@!OIDF.OID4VCI]. If batch issuance is used and the Credential Issuer has indicated (via `cryptographic_binding_methods_supported ` in it's metadata) that cryptographic binding is required, all public keys used in Credential Request SHOULD be attested within a single key attestation.

Reviewer 2: ```suggestion
Wallets MUST support key attestations as defined in Annex D of [@!OIDF.OID4VCI]. If batch issuance is used and the Credential Issuer has indicated (via `cryptographic_binding_methods_supported ` metadata parameter) that cryptographic key binding is required, all public keys used in Credential Request SHOULD be attested within a single key attestation.
```

| iss | MUST |[@!RFC7519], Section 4.1.1 |
| iat | MUST |[@!RFC7519], Section 4.1.6 |
| exp | SHOULD (at the discretion of the Issuer) | [@!RFC7519], Section 4.1.4 |

> | cnf |	MUST (if credential has cryptographic holder binding) |	[@!RFC7800]|

Reviewer: ```suggestion
| cnf |	MUST (if credential has cryptographic key binding) |	[@!RFC7800]|
```

### Cryptographic Holder Binding between VC and VP

> * If the credential has cryptographic holder binding, a KB-JWT, as defined in [@!I-D.ietf-oauth-sd-jwt-vc], MUST always be present when presenting an SD-JWT VC.

Reviewer: ```suggestion
* If the credential has cryptographic key binding, a KB-JWT, as defined in [@!I-D.ietf-oauth-sd-jwt-vc], MUST always be present when presenting an SD-JWT VC.
```

   [[ To be removed from the final specification ]]

   -05

>    * Added support for credentials without cryptographic holder binding

Reviewer: ```suggestion
   * Added support for credentials without cryptographic key binding
```
